### PR TITLE
update the README according to OCA guidelines

### PR DESCRIPTION
**Pull request by EvvFoxy**

### CODE REVIEW

**Review by luc-demeyer — APPROVED** [review]

**Comment by sbidoul** [comment]
Hi Els and Luc,

The README fragments must have only one subtitle level, and the tilde character must be used as underline.

**Comment by sbidoul** [comment]
Also removes the titles from DESCRIPTION.rst. They will be added by the template.

File: role_policy/readme/ROADMAP.rst [hunk 1]
```diff
@@ -0,0 +1,15 @@
+=======
+Roadmap
+=======
```
Comment (sbidoul) [inline role_policy/readme/ROADMAP.rst]: remove this, titles are added by the template

File: role_policy_account/readme/DESCRIPTION.rst [hunk 2]
```diff
@@ -0,0 +1,5 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
```
Comment (sbidoul) [inline role_policy_account/readme/DESCRIPTION.rst]: Remove this, the badges are added by the template

File: role_policy_sale/readme/DESCRIPTION.rst [hunk 3]
```diff
@@ -0,0 +1,5 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
```
Comment (sbidoul) [inline role_policy_sale/readme/DESCRIPTION.rst]: remote this badge

File: role_policy/readme/DESCRIPTION.rst [hunk 4]
```diff
@@ -0,0 +1,209 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
```
Comment (sbidoul) [inline role_policy/readme/DESCRIPTION.rst]: remove this badge

**Review by sbidoul — COMMENTED** [review]
Also, remove the `.DS_Store` files

**Pull request merged**